### PR TITLE
feature/PN-7255

### PR DESCRIPTION
**Pull request by fdammassa**

### CODE REVIEW

File: docs/openapi/api-external-b2b-csost.yaml [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+            - se il file identificato da fileKey non è ancora stato salvato; <br/>
+
+  
+  ################################################################################################
+  ###                                AGGIORNA STATO CONSERVAZIONE FILE                         ###
+  ################################################################################################
+  '/cn/v1/events':
+    post:
+      operationId: updateLegalConservationEvents
+      'x-pagopa-lambda-name': 'pn-eventUpdateRecevier'
```
Comment (ViVeniero) [inline docs/openapi/api-external-b2b-csost.yaml:111]: pn-eventUpdateRecevier -> pn-eventUpdateReceiver

File: docs/openapi/api-external-b2b-csost.yaml [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+            Formattata con timezone secondo l'rfc 3339
+        isFinal:
+          type: boolean
+          description: True se lo stato è finale e non sono possibili ulteriore aggiornamenti sullo stesso documento
+
+    OperationResultCodeResponse:
+      required:
+        - resultCode
+        - resultDescription  
+        - clientResponseTimeStamp
```
Comment (ViVeniero) [inline docs/openapi/api-external-b2b-csost.yaml:265]: Manca in properties?

File: docs/openapi/aws/api-cn-CN_BE-aws.yaml [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+        isFinal:
+          type: boolean
+          description: >-
+            True se lo stato è finale e non sono possibili ulteriore
+            aggiornamenti sullo stesso documento
+    OperationResultCodeResponse:
+      required:
+        - resultCode
+        - resultDescription
+        - clientResponseTimeStamp
```
Comment (ViVeniero) [inline docs/openapi/aws/api-cn-CN_BE-aws.yaml:279]: Manca in properties?

File: docs/openapi/api-internal-b2b-csost.yaml [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+            Formattata con timezone secondo l'rfc 3339
+        isFinal:
+          type: boolean
+          description: True se lo stato è finale e non sono possibili ulteriore aggiornamenti sullo stesso documento
+
+    OperationResultCodeResponse:
+      required:
+        - resultCode
+        - resultDescription  
+        - clientResponseTimeStamp
```
Comment (ViVeniero) [inline docs/openapi/api-internal-b2b-csost.yaml:265]: Manca in properties?

**Review by ViVeniero — COMMENTED** [review]
Vedi commenti

File: docs/openapi/api-internal-b2b-csost.yaml (hunk 4 above)
Comment (fdammassa) [reply docs/openapi/api-internal-b2b-csost.yaml]: @ViVeniero rimosso clientResponseTimeStamp dai required fields

File: docs/openapi/aws/api-cn-CN_BE-aws.yaml (hunk 3 above)
Comment (fdammassa) [reply docs/openapi/aws/api-cn-CN_BE-aws.yaml]: @ViVeniero rimosso clientResponseTimeStamp dai required fields

File: docs/openapi/api-external-b2b-csost.yaml (hunk 2 above)
Comment (fdammassa) [reply docs/openapi/api-external-b2b-csost.yaml]: @ViVeniero rimosso clientResponseTimeStamp dai required fields

**Review by ViVeniero — APPROVED** [review]

**Pull request merged**